engine/src/main/java/org/terasology/world/block/tiles/WorldAtlasImpl.java

```diff
@@ -205,7 +224,7 @@ private void buildAtlas() {
 
         int numMipMaps = getNumMipmaps();
         ByteBuffer[] data = createAtlasMipmaps(numMipMaps, TRANSPARENT_COLOR, tiles, "tiles.png");
-        ByteBuffer[] dataNormal = createAtlasMipmaps(numMipMaps, UNIT_Z_COLOR, tilesNormal, "tilesNormal.png");
+        ByteBuffer[] dataNormal = createAtlasMipmaps(numMipMaps, UNIT_Z_COLOR, tilesNormal, "tilesNormal.png", tilesGloss);
```

Comment:
Agreed,  there is definitely some reorganization needed with this stuff.  I hope that a better strategy to accommodate net new features like environmental reflections will come about when this gets an overhaul.  It seems extremely involved to add textures that are shared between shaders at the moment,  hopefully this will become easier in the future.